rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -476,36 +471,19 @@ public void onBrowserEvent(Event event) {
 
                     MenuItem removeItem = new MenuItem("Remove",
                                                        RMImages.instance.node_remove_16().getSafeUri().asString());
-                    removeItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-                        @Override
-                        public void onClick(MenuItemClickEvent event) {
-                            controller.removeNodes();
-                        }
-                    });
+                    removeItem.addClickHandler(event15 -> controller.removeNodes());
 
                     MenuItem lockItem = new MenuItem("Lock", lockItemImageResource);
-                    lockItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-                        @Override
-                        public void onClick(MenuItemClickEvent event) {
-                            controller.lockNodes();
-                        }
-                    });
+                    lockItem.addClickHandler(event14 -> controller.lockNodes());
 
                     MenuItem unlockItem = new MenuItem("Unlock", unlockItemImageResource);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)